res/css/_spacing.pcss

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 /* 1rem :: 10px */
 
 $spacing-4: 4px;
+$spacing-6: 6px;
```

Comment:
Hmm, are you implementing new designs that use a 6px spacing? Compound does not have this as a standard spacing https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=1336%3A18203 so it shouldn't be added as a variable here.